docs/topics/http/sessions.txt

```diff
@@ -348,6 +348,16 @@ Bundled serializers
         >>> request.session['0']
         'bar'
 
+
+    By default your serializer in Django does not allow non-UTF8 bytes to be stored in session values, note that using non unicode values for ``request.session`` won't work as expected::
```

Comment:
I'm not sure a full example is needed. I think a single sentence like "Similarly, data that can’t be encoded in JSON, such as non-UTF8 bytes like '\xd9', can’t be stored." might be enough. Any thoughts, @erikr?